node-stringprep.cc

```diff
@@ -55,27 +55,27 @@ class StringPrep : public ObjectWrap {
           }
         catch (UnknownProfileException &)
           {
-            NanThrowTypeError("Unknown StringPrep profile");
-            NanReturnUndefined();
+            Nan::ThrowTypeError("Unknown StringPrep profile");
+            return;
           }
 
         StringPrep *self = new StringPrep(profileType);
         if (self->good())
           {
-            self->Wrap(args.This());
-            NanReturnValue(args.This());
+            self->Wrap(info.This());
+            info.GetReturnValue().Set(info.This());
```

Comment:
`return` - not strictly necessary but you're returning in other branches so be safe and communicate intention to future devs